client/rpc/src/testing.rs

```diff
@@ -20,11 +20,50 @@
 
 use std::{future::Future, sync::Arc};
 
-use sp_core::testing::TaskExecutor;
+/// A task executor that can be used for running RPC tests.
+///
+/// Warning: the tokio runtime must be initialized before calling this.
+#[derive(Clone)]
+pub struct TokioTestExecutor(tokio::runtime::Handle);
```

Comment:
jsonrpsee is now using tokio channels so we must run the tests on the tokio runtime to work..